client/webserver/site/src/js/markets.ts

```diff
@@ -1659,6 +1659,33 @@ export default class MarketsPage extends BasePage {
     page.vPreorderErrTip.dataset.tooltip = msg
   }
 
+  showPreOrderAdvancedOptions (order: TradeForm, swap: PreSwap, redeem: PreRedeem, changed: ()=>void) {
+    const page = this.page
+    Doc.hide(page.showAdvancedOptions)
+    Doc.show(page.hideAdvancedOptions)
+    this.reloadOrderOpts(order, swap, redeem, true, changed)
+  }
+
+  hidePreOrderAdvancedOptions (order: TradeForm, swap: PreSwap, redeem: PreRedeem, changed: ()=>void) {
+    const page = this.page
+    Doc.hide(page.hideAdvancedOptions)
+    Doc.show(page.showAdvancedOptions)
+    this.reloadOrderOpts(order, swap, redeem, false, changed)
```

Comment:
We shouldn't need this. What were you seeing that required a reload?

Reply:
Right now we only keep `page.vOrderOpts` in memory, so I have no way to know if it should be showed by default or not if I only hide or show the child node

Reply:
I see now. A preferable pattern would be to create and insert all of the elements, and selectively `Doc.{hide,show}` them based on the user's selection.

Reply:
that user selection would be some settings like "show by default" with a checkbox? What do you have in mind?

Reply:
check out https://github.com/buck54321/dcrdex/compare/97f7f3ab9ed3a05282e7098d637e9af9d2c28fa9...buck54321:dcrdex:defaultopts-alt

Reply:
Thanks for the suggestion, pushed it on my PR